interpreter/text/arrange.ml

```diff
@@ -448,11 +448,7 @@ let assertion mode ass =
   | AssertUninstantiable (def, re) ->
     Node ("assert_trap", [definition mode None def; Atom (string re)])
   | AssertReturn (act, lits) ->
-    Node ("assert_return", action act :: List.map literal lits)
-  | AssertReturnCanonicalNaN act ->
-    Node ("assert_return_canonical_nan", [action act])
-  | AssertReturnArithmeticNaN act ->
-    Node ("assert_return_arithmetic_nan", [action act])
+    Node ("assert_return", [action act]) (*  TODO :: List.map literal lits) *)
```

Comment:
This will need to handle `modifier` (or whatever we want to call it) instead of a list of literals but, to make sure, this is a serializer of the AST?